lms/envs/common.py

```diff
@@ -2562,6 +2562,10 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
     # Management of per-user schedules
     'openedx.core.djangoapps.schedules',
     'rest_framework_jwt',
+
+    # ADG Features
```

Comment:
How about changing comment to ADG Apps

Reply:
I thought so as well, but it will be redundant because the name of the list which contains this is `INSTALLED_APPS`

Reply:
How about adding a new list `ADG_INSTALLED_APPS` and then extending `INSTALLED_APPS` with this new list? It will define a more specific group as compare to the group with just a comment. It will also help, enforcing everyone to register new apps at single place. Also it would be easier for us to move code in a separate project.